lib/cadet/assessments/assessments.ex

```diff
@@ -850,7 +954,7 @@ defmodule Cadet.Assessments do
          {:status, :submitted} <- {:status, submission.status},
          {:allowed_to_unsubmit?, true} <-
            {:allowed_to_unsubmit?,
-            role == :admin or bypass or
+            role == :admin or bypass or is_nil(submission.student_id) or
```

Comment:
Why was this condition added?

Reply:
This is to allow all avengers to unsubmit team submissions. The concept of group does not exist here.